src/table.js

```diff
@@ -492,6 +475,24 @@ export function __table(source, operations) {
     const [{value: column}] = operands;
     const values = operands.slice(1).map(({value}) => value);
     switch (type) {
+      // valid (matches the column type)
+      case "v": {
+        const [colType] = values;
+        const filter = colType === "date" ?
+          (d) => (d[column] instanceof Date) && !Number.isNaN(+d[column]) :
+          (d) => typeof d[column] === colType;
```

Comment:
TODO

* For the `date` type, stylistic simplification: `!isNaN(d[column])`
* For the `number` type, check isNaN too
* For the `buffer` type, this should check `d[column] instanceof ArrayBuffer`?
* For the `array` type, this should check `Array.isArray(d[column])`?
* For the `object` type, we’ll need to exclude null (because `typeof null` is `"object"`)
* For the `other` type, this should just check not-nullish?

Reply:
Done in the last commit ✅ !